### ISSUE TITLE
Pie Chart and Ring Chart - Make Labeling Clearer

### ISSUE THREAD

**Issue by acell** [SOCR/SOCRAT-issues#66]
While only minorly gross when there are only a few categories, with many categories it is very difficult to read the pie / ring charts. Either making a legend with labels or putting the labels outside of the circle would be better.

<img width="613" alt="screen shot 2017-02-02 at 11 23 26 am" src="https://cloud.githubusercontent.com/assets/17788969/22557960/11bb6efe-e93a-11e6-8c07-1a850624b374.png">
<img width="597" alt="screen shot 2017-02-02 at 11 23 21 am" src="https://cloud.githubusercontent.com/assets/17788969/22557959/11bac9c2-e93a-11e6-81a7-ad39b7f9bf15.png">

**Comment by acell** [comment]
This is different on dev mode, but is half in the pie chart and half outside of the pie chart, which is still not ideal. Adding titles (as the other graphs at least have axes showing what is being measured) would also be nice for clarity. 

<img width="609" alt="screen shot 2017-02-02 at 11 40 34 am" src="https://cloud.githubusercontent.com/assets/17788969/22558666/7f19d146-e93c-11e6-8c54-a259ca768bf8.png">

**Comment by acell** [comment]
Added a li'l conditional loop to check whether a label is on the left side and then if so it puts the label at a radius further from the circle. Limitations include that the view box was v small so I also ended up rotating the labels on the y-axis down a tad so that the iris top label would remain insight. This reveals itself in a not super pretty way when there are a shootload of categories, observed in the second screenshot. Otherwise it pretty much does the basic thing it should do, but accounting for the length of a label and then adjusting the view accordingly would be a more complex and also better way to do this. Unless we just added a legend. Welp. Didn't make a pull request for this (yet / because it isn't overly polished and also it still has commits from previous pull request) but did commit it to my fork.

<img width="644" alt="screen shot 2017-02-09 at 3 44 15 pm" src="https://cloud.githubusercontent.com/assets/17788969/22802617/74627cbe-eedf-11e6-8528-a207a2c2e7bc.png">
<img width="744" alt="screen shot 2017-02-09 at 3 48 14 pm" src="https://cloud.githubusercontent.com/assets/17788969/22802622/7a5c771e-eedf-11e6-83d5-302daa3d491f.png">
